Tasks/VsTestV2/nondistributedtest.ts

```diff
@@ -105,18 +98,17 @@ export class NonDistributedTest {
             return 1;
         }
     }
-
-    private getTestAssemblies(): string[] {
-        tl.debug('Searching for test assemblies in: ' + this.inputDataContract.TestSelectionSettings.SearchFolder);
-        return tl.findMatch(this.inputDataContract.TestSelectionSettings.SearchFolder, this.sourceFilter);
-    }
-
     private createTestSourcesFile(): string {
         try {
             console.log(tl.loc('UserProvidedSourceFilter', this.sourceFilter.toString()));
-
+            const telemetryProps: { [key: string]: any; } = { MiniMatchLines: this.sourceFilter.length };
```

Comment:
can you add one more property called 'ExecutionFlow' with value 'Distributed' and 'NonDistributed' based on the file you made the changes in? Sorry i forgot to tell this in the beginning